README.md

```diff
@@ -1,64 +1,55 @@
 
 # PHPTAL - Template Attribute Language for PHP
 
-Master: [![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
+[![Monthly Downloads](https://poser.pugx.org/phptal/phptal/d/monthly)](https://packagist.org/packages/phptal/phptal)
+[![License](https://poser.pugx.org/phptal/phptal/license)](LICENSE)
+[![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
+[![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/phptal/PHPTAL/badges/quality-score.png)](https://scrutinizer-ci.com/g/phptal/PHPTAL/)
 
-Usage requirements
-==================
+Requirements
+============
 
 To use PHPTAL in your projects, you will only require PHP 5.1.2 or later.
 
-If you want to use the builtin internationalisation system (I18N) the gettext extension must be compiled into PHP (`--with-gettext`).
+If you want to use the builtin internationalisation system (I18N), the php-gettext extension must be installed or compiled into PHP (`--with-gettext`).
+
+Composer install (recommended)
+==============================
 
+You can install this package by using [Composer](http://getcomposer.org).
+Link to Packagist: https://packagist.org/packages/phptal/phptal
 
-Non-PEAR install
-================
+```
+{
+    "require": {
+        "phptal/phptal": "~1.3"
+    }
+}
+```
```

Comment:
Editing the `composer.json` file is discouraged. Use `composer require phptal/phptal` instead.